mmv1/templates/terraform/examples/Storage_pool_create.tf.erb

```diff
@@ -4,5 +4,5 @@ resource "google_netapp_storage_pool" "<%= ctx[:primary_resource_id] %>" {
   location = "us-central1"
   service_level = "PREMIUM"
   capacity_gib = "2048"
-  network = "projects/{{project_number}}/global/networks/cxo-automation-vpc-01"
+  network = ""
```

Comment:
It'd be nice if you could create a network in this config and then use it in the resource. See examples like https://github.com/GoogleCloudPlatform/magic-modules/blob/5a1405f4cf4718306b4645e657964aacf4cfdf24/mmv1/templates/terraform/examples/router_nat_basic.tf.erb#L1